scripts/forward-bash/forward.sh

```diff
@@ -0,0 +1,285 @@
+        "staging") echo "Dialogporten-Staging" ;;
+        "prod") echo "Dialogporten-Prod" ;;
+        *) echo "" ;;
+    esac
+}
+
+# Logging functions
+log_info() {
+    echo -e "${BLUE}ℹ${NC} $1"
+}
+
+log_success() {
+    echo -e "${GREEN}✓${NC} $1"
+}
+
+log_warning() {
+    echo -e "${YELLOW}⚠${NC} $1"
+}
+
+log_error() {
+    echo -e "${RED}✖${NC} $1" >&2
+}
+
+log_title() {
+    echo -e "\n${BOLD}${CYAN}$1${NC}"
+}
+
+print_box() {
+    local title="$1"
+    local content="$2"
+    local width=55
+    
+    # Top border
+    printf "╭%${width}s╮\n" | tr ' ' '─'
+    
+    # Title line with proper padding
+    printf "│ ${BOLD}%s${NC}%$((width - ${#title} - 1))s│\n" "$title"
+    
+    # Empty line
+    printf "│%${width}s│\n" ""
+    
+    # Content (handle multiple lines)
+    while IFS= read -r line; do
+        # Remove escape sequences for length calculation
+        clean_line=$(echo -e "$line" | sed 's/\x1b\[[0-9;]*m//g')
+        # Calculate padding needed
+        padding=$((width - ${#clean_line} - 2))
+        # Print line with proper padding
+        printf "│ %b%${padding}s│\n" "$line" " "
+    done <<< "$content"
+    
+    # Bottom border
+    printf "╰%${width}s╯\n" | tr ' ' '─'
+}
+
+# Check prerequisites
+check_dependencies() {
+    if ! command -v az >/dev/null 2>&1; then
+        log_error "Azure CLI is not installed. Please visit: https://docs.microsoft.com/en-us/cli/azure/install-azure-cli"
+        exit 1
+    fi
+    log_success "Azure CLI is installed"
+}
+
+get_subscription_id() {
+    local env=$1
+    local subscription_name
+    subscription_name=$(get_subscription_name "$env")
+    
+    if [ -z "$subscription_name" ]; then
+        log_error "Invalid environment: $env"
+        exit 1
+    fi
+    
+    local sub_id
+    sub_id=$(az account show --subscription "$subscription_name" --query id -o tsv 2>/dev/null)
+    
+    if [ -z "$sub_id" ]; then
+        log_error "Could not find subscription '$subscription_name'. Please ensure you are logged in to the correct Azure account."
+        exit 1
+    fi
+    
+    echo "$sub_id"
+}
+
+get_postgres_info() {
+    local env=$1
+    local subscription_id=$2
+    
+    local name
+    name=$(az postgres flexible-server list --subscription "$subscription_id" \
+        --query "[?tags.Environment=='$env' && tags.Product=='Dialogporten'] | [0].name" -o tsv)
+    
+    if [ -z "$name" ]; then
+        log_error "Postgres server not found"
+        exit 1
+    fi
+    
+    local hostname="${name}.postgres.database.azure.com"
+    local port=5432
+    
+    local username
+    username=$(az postgres flexible-server show \
+        --resource-group "dp-be-${env}-rg" \
+        --name "$name" \
+        --query "administratorLogin" -o tsv)
+    
+    echo "name=$name"
+    echo "hostname=$hostname"
+    echo "port=$port"
+    echo "connection_string=postgresql://${username}:<retrieve-password-from-keyvault>@${hostname}:${port}/dialogporten"
```

Comment:
Dette er connection string som skal brukes lokalt? Blir det ikke `localhost` her da? 
```suggestion
    echo "connection_string=postgresql://${username}:<retrieve-password-from-keyvault>@localhost:${port}/dialogporten"
```

Reply:
ja sant, det stemmer!